### PR TITLE
Move copyright header handling explicitly into the repo

### PR DESCRIPTION
**Pull request by freben**

### CODE REVIEW

**Review by Rugvip — APPROVED** [review]
:shipit:

File: .changeset/orange-walls-reply.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': patch
+---
+
+Depend on eslint-plugin-notice directly instead of via backstage OSS, in preparation for copyright header change
```
[the remark this answers is not in the document]
Comment (freben) [reply .changeset/orange-walls-reply.md]: aye! updated

File: packages/cli/package.json [hunk 2]
```diff
@@ -71,6 +71,7 @@
     "eslint-plugin-jest": "^24.1.0",
     "eslint-plugin-jsx-a11y": "^6.2.1",
     "eslint-plugin-monorepo": "^0.3.2",
+    "eslint-plugin-notice": "^0.9.10",
```
Comment (Rugvip) [inline packages/cli/package.json:74]: Eh, realizing that we shouldn't be shipping this with the CLI, it should be installed in the root of the repo instead. This check is specific to the main repo only

**Review by Rugvip — CHANGES_REQUESTED** [review]

File: packages/cli/package.json (hunk 2 above)
Comment (freben) [reply packages/cli/package.json]: Yeah that's totally right, good we didn't merge this before fixing that :)

File: package.json [hunk 3]
```diff
@@ -60,6 +60,7 @@
     "@spotify/prettier-config": "^9.0.0",
     "command-exists": "^1.2.9",
     "concurrently": "^6.0.0",
+    "eslint-plugin-notice": "^0.9.10",
```
Comment (Rugvip) [inline package.json:63]: Get rid of the `@spotify/eslint-config-oss` now ye? 😅

**Review by Rugvip — APPROVED** [review]
Alright, actually :shi

**Review by Rugvip — CHANGES_REQUESTED** [review]
I not good so GitHub

File: package.json (hunk 3 above)
Comment (freben) [reply package.json:63]: Um do I have to dig through it to see what other things it provides on top of our own cli stuff and copy over that as well? :/

Comment (Rugvip) [reply package.json:63]: There isn't a lot of digging to do, this is all it does: https://github.com/spotify/eslint-config-oss/blob/master/index.js

**Review by Rugvip — APPROVED** [review]

**Pull request merged**